src/System.Xml.XDocument/tests/xNodeBuilder/CommonTests.cs

```diff
@@ -3400,7 +3401,9 @@ public void CData_4()
                 }
 
                 //[Variation(Id = 5, Desc = "WriteCData with ]]>", Priority = 1)]
-                public void CData_5()
+                [Fact]
+                [ActiveIssue(4054, default(PlatformID))]
```

Comment:
What is the ```default(PlatformID)``` for?  Is the goal to disable this for all platforms?  If so, then just do ```[ActiveIssue(4054)]```... if instead the goal is to disable it for a specific platform, then you can use ```[ActiveIssue(4054, PlatformID.PlatformName)]```.